build.project.xml

```diff
@@ -76,26 +76,20 @@
         <phingcall target="disable-persistent-session-cookies" />
     </target>
 
-    <target name="setup-virtuoso-permissions">
```

Comment:
The `setup-virtuoso-permissions` target is being moved to the `ec-europa/virtuoso` package, but that means it should get a release and we should depend on the new release on our `composer.json`.

Reply:
Ignore this remark, having the patches is good for now.